### ISSUE TITLE
Question about reproduction on 7Scenes

### ISSUE THREAD

**Issue by tdd233** [kaichen-z/VMLoc#5]
Hi, Kaichen
Thanks for the code, I'm very interested in your outstanding work!!!
I tried to load your trained models (redkitchen) which is already provided, but the inference result is lower than the result from your paper. So I want to ask you what might cause this situation.
****
result from your paper:  0.21m, 5.01°
result from my inference by loading your weights:  0.18m, 5.52°
****
By the way, the pytorch version I used is 1.10.1, which is the same as the version you mentioned.
****
python==3.8 
pytorch==1.10.1 
torchvision==0.11.2 
****
Looking forward to your reply!

**Comment by kaichen-z** [comment]
Dear Tdd,

Thank you for your interest. Please note that VMLoc's performance may be affected by minor variances. In the provided checkpoint, the translation accuracy is better than what we reported in the original paper, but the rotation accuracy is somewhat lower.

Best regards,

**Comment by tdd233** [comment]
Dear Kaichen,

Thanks for your reply. I tried to retrain the network using pytorch version of 1.10.1, but the reproduced result is lower than result from the provided checkpoint, which is 0.19m, 6.86°. However, when I use the pytorch version of 0.4.1 to retrain the network, the reproduced result is 0.18m, 5.52°. But the provided checkpoint seems to be trained on pytorch version of 1.10.1. Do you know the reason? And should I explore further work by using the pytorch version of 0.4.1?

Looking forward to your reply!
